docs/components/buttons/android.md

```diff
@@ -0,0 +1,113 @@
+
+
+## Syntax
+
+```kotlin example
+@Composable
+fun WarpButton(
+    text: String, 
+    onClick: () -> Unit, 
+    modifier: Modifier = Modifier, 
+    enabled: Boolean = true, 
+    buttonStyle: WarpButtonStyle, 
+    maxLines: Int = 1, 
+    loading: Boolean = false
+)
+```
+
+## Visual options
+
+### Primary
+
+The primary button is a call to action. As a general rule, there should only be
+one of them on the screen. This guides the user towards the happy path.
+
+```kotlin example
+WarpButton(
+    onClick = { }, 
+    buttonStyle = WarpButtonStyle.Primary, 
+    text = "Save"
+)
+```
+
+### Secondary
+
+Secondary buttons are without background, and are often used for secondary actions. Also available as `quiet` style.
+
+```kotlin example
+WarpButton(
+    onClick = { }, 
+    buttonStyle = WarpButtonStyle.Secondary, 
+    text = "Save"
+)
+WarpButton(
+    onClick = { }, 
+    buttonStyle = WarpButtonStyle.Quiet, 
+    text = "Save"
+)
+```
+
+### Negative
+
+Used for destructive actions, like deletion. Shouldn't be used on the same
+screen as a primary button.
+
+```kotlin example
+WarpButton(
+    onClick = { }, 
+    buttonStyle = WarpButtonStyle.Critical, 
+    text = "Cancel"
+)
+```
+
+### Loading/In progress
+
+Used for visual feedback that the action the user triggered is loading.
+
+```kotlin example
+var loading by remember { mutableStateOf(false) }
+var buttonText by remember { mutableStateOf("Save") }
+val onClickAction = { }
+
+WarpButton(
+    onClick = onClickAction, 
+    buttonStyle = WarpButtonStyle.Primary, 
+    text = buttonText,
+    loading = loading
+)
+```
+
+### Disabled
+
+Disabled is an anti-pattern and is not supported. There will ALWAYS be users who
+don't understand why an element is disabled, or users who can't even see that it
+is disabled because of poor lighting conditions or other reasons.
+
+```kotlin example
+var enabled by remember { mutableStateOf(false) }
+
+WarpButton(
+    onClick = { }, 
+    buttonStyle = WarpButtonStyle.Secondary, 
+    text = "Edit",
+    enabled = enabled
+)
```

Comment:
Question only for my understanding 😊 Should this be button be styled as disabled? Can't see anything that changes styling to disabled in the example 🤔

Reply:
The button can be set as disabled, but it needs a default value which should be `enabled` unless stated otherwise